### ISSUE TITLE
Code for SLIC loss?

### ISSUE THREAD

**Issue by sunshineatnoon** [fuy34/superpixel_fcn#10]
Hi, Thanks for open-sourcing this awesome work. I noticed that this repo is for the segmentation and depth estimation tasks. Could you also provide the code or hyper-parameters for Eq.(4) in the paper? Thanks!

**Comment by fuy34** [comment]
Hi, 

Thank you for your interest in our work. The ```m``` is set to be 0.003 for segmentation (with semantic loss) and 30/60 (with color loss) for stereo matching, and ```S``` is the cell size based on your choice. 

More details can be found in the "implement details" and "Application to Stereo Matching" sections of the paper.  And you can modify them here in the code.  
https://github.com/fuy34/superpixel_fcn/blob/0f07be910e41ac73404de23caa99f0e454c6440c/main.py#L68

**Comment by sunshineatnoon** [comment]
Thanks for the quick reply! Is it possible to share the code for the SLIC loss?

**Comment by sunshineatnoon** [comment]
I tried to implement this loss, but observe many small super pixels during training. I used the CIELab image feature and didn't use the semantic masks or depth during training. The visualization can be found [here](https://www.dropbox.com/s/7tt7ftilgvu6ilj/individualImage.png?dl=0).

**Comment by fuy34** [comment]
I see.  The BSDS500 dataset is a little noisy in the color domain. 
You may need to perform bilateral filtering or any other filtering method you like before sending them to the network. For the stereo matching task, the images are cleaner and no need for this step. 
Good luck!

**Comment by sunshineatnoon** [comment]
Got it. Thanks so much!
